app/assets/stylesheets/style.css

```diff
@@ -585,7 +604,7 @@ textarea, input {
   background: rgba(255,255,255,0.8);
   z-index: 9999;
   border-radius: 4px;
-  padding: 10px; 
+  padding: 10px;
```

Comment:
Please remove lines having no change from pr. I mean reviewer should only see lines which require attention. This saves ambiguity and wrong merges

Reply:
My editor removes blank spaces, I thought it was redundant to have blank spaces.
That's why I removed them.

Reply:
I'll add them again :)

Reply:
If you want some refactoring OR reformating changes then create a separate pr for them. Kindly don't include them in functional prs

Reply:
Got it. Thanks.